src/calculateRentalCost.js

```diff
@@ -3,8 +3,30 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_CAR_C0ST = 40;
+
+  const LONG_TERM = 7;
+  const LONG_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_DISCOUNT = 20;
+
+  let allResult = 0;
+
+  if (days >= LONG_TERM) {
+    allResult += RENT_CAR_C0ST * days - LONG_DISCOUNT;
+
+    return allResult;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    allResult += RENT_CAR_C0ST * days - SHORT_DISCOUNT;
+
+    return allResult;
```

Comment:
The check 'days < LONG_TERM' is redundant because if 'days' was greater than or equal to 'LONG_TERM', the previous 'if' condition would have already returned a value.